docker-compose.yml

```diff
@@ -1,18 +1,24 @@
-version: "2.0"
+version: '3'
 services:
   titra:
     image: kromit/titra
-    container_name: titra
+    container_name: titra_app
```

Comment:
better name for containers

Reply:
I also agree on this one but wonder if this is also a breaking change for people who depend on the container name for their setup - is this a common convention to use _app for the container name? If yes I would take the chance of the next major release to change this as well.

Reply:
Yeah for example the db was name `mongodb` but many projects ships a mongodb image so if they all name the same this is problematic. So instead the convention in docker-compose is to name the containers `PROJECTNAME_CONTAINERROLE`. So in this case `titra_app`, `titra_db`, etc.